README.md

```diff
@@ -1,15 +1,133 @@
+- Copy the value of the auto-generated key, or create a new one with a name that's meaningful to you
+
+
+![Example of how to get your private key](https://cdn.builder.io/api/v1/image/assets%2Fd1ed12c3338144da8dd6b63b35d14c30%2Fd824c5a4ddc3436e806d4b7305961d96)
+
+![Example of how to get your private key](https://cdn.builder.io/api/v1/image/assets%2Fd1ed12c3338144da8dd6b63b35d14c30%2F7a121a72a0f749a1a9d154b96914be26)
+
+
+## Initialize a space with this starter kit
+Follow the steps below to initialize a blank Builder.io space with the starter kit features and config.
+#### *Note: If you want to clone/import our pre created templates for models and custom targeting, see the Cloning/Importing section below and then come back to this section.*
+
+#### Clone this repository and initialize a Builder.io space
+
+Next, we'll create a copy of the starter project, and create a new
+[space](https://www.builder.io/c/docs/spaces) for it's content to live
+in.
+
+In the example below, replace `<private-key>` with the key you copied
+in the previous step, and change `<space-name>` to something that's
+meaningful to you -- don't worry, you can change it later!
+
+Note:
+if you're only interested in using this starter for a landing page use this command instead:
+
+```
+builder create --key "<private-key>" --name "<space-name>" --input builder-landing-page-only --debug
+```
+
+
+If this was a success you should be greeted with a message that
+includes a public API key for your newly minted Builder.io space.
+
+*Note: This command will also publish some starter builder.io cms
+content from the ./builder directory to your new space when it's
+created.*
+
+``` bash
+  ____            _   _       _                     _                    _   _ 
+| __ )   _   _  (_) | |   __| |   ___   _ __      (_)   ___       ___  | | (_)
+|  _ \  | | | | | | | |  / _` |  / _ \ | '__|     | |  / _ \     / __| | | | |
+| |_) | | |_| | | | | | | (_| | |  __/ | |     _  | | | (_) |   | (__  | | | |
+|____/   \__,_| |_| |_|  \__,_|  \___| |_|    (_) |_|  \___/     \___| |_| |_|
+
+|████████████████████████████████████████| product-footer writing schema.json | 1/1
+|████████████████████████████████████████| announcement-bar: writing schema.json | 1/1
+|████████████████████████████████████████| category-hero: writing schema.json | 1/1
+|████████████████████████████████████████| page: writing schema.json | 2/2
+
+
+Your new space "sfcc pwa kit starter" public API Key: d1ed12c3338144da8dd6b63b35d14c30
+```
+
+Copy the public API key ("d1ed12c3338144da8dd6b63b35d14c30" in the example above) for the next step.
+
+This starter project uses dotenv files to configure environment variables.
+Open the files [.env.development](./.env.development) and
+[.env.production](./.env.production) in your favorite text editor, and
+set the value of `REACT_APP_BUILDER_PUBLIC_KEY` to the public key you just copied.
+You can ignore the other variables for now, we'll set them later.
+
+```diff
```

Comment:
No env files in this repo, it should be this file https://github.com/BuilderIO/sfcc-pwa-kit-starter/blob/main/app/utils/builder.js